stories/RangeInput.stories.js

```diff
@@ -14,4 +14,22 @@ storiesOf('RangeInput', module)
   }))
   .add('with min and max', () => ({
     template: `<ais-range-input attribute="price" :min=10 :max=50></ais-range-input>`,
+  }))
+  .add('using the default slot', () => ({
+    template: `
+      <ais-range-input attribute="price">
+        <template slot-scope="{refine, updateMin, updateMax, currentRefinements}">
+          <form  @submit.prevent="refine()" >
```

Comment:
hmm, I don't understand why it has no arguments here

Reply:
The update and the search are done in two steps. I use updateMin and updateMax to update the value and refine to refine the search state with the updated values and trigger the search.